### ISSUE TITLE
Keep getting message "Couldn't load sound/honey/ting1.wav"

### ISSUE THREAD

**Issue by cunningmunki** [vittorioromeo/quakevr#46]
Once I start a game I'm being continually spammed by messages that read "couldn't load sound/honey/ting1.wav" then "couldn't load sound/honey/ting2.wav", followed by "...ting3.wav" and "...ting4.wav", over and over.

There isn't even a "sound" folder in my Quake dir!

**Comment by Spacedg** [comment]
Same thing here.

**Comment by Glidos** [comment]
+1

**Comment by Glidos** [comment]
The missing files are here: https://www.quaddicted.com/reviews/honey.html.

I'm using just the sound folder, placing it in Quake\Id1, so as to have

Quake\Id1\sound\honey\ting1,wav etc.

**Comment by cunningmunki** [comment]
> The missing files are here: https://www.quaddicted.com/reviews/honey.html.
> 
> I'm using just the sound folder, placing it in Quake\Id1, so as to have
> 
> Quake\Id1\sound\honey\ting1,wav etc.

Thanks, I'll give it a try!!!

But I wonder why it's expecting a sound file from some random community map?!

**Comment by Glidos** [comment]
Agreed. That is very strange. Perhaps it isn't the correct file at all, but then it would be an even stranger coincidence.

**Comment by SuperV1234** [comment]
> But I wonder why it's expecting a sound file from some random community map?!

I hardcoded `honey` support as part of the VR `progs` file, as it was one of the highest rated community maps. That's the reason why the sound is required. I went down the route of making a "mega `progs`" file with hardcoded support for multiple mods at the same time, as the changes I made to add VR features were not compatible with mods out of the box.

The correct fix here is to suppress the warning, check what wants to play `ting1.wav`, and only play that sound if we are in a `honey` map.
